smarts/core/tests/test_trajectory_controller.py

```diff
@@ -40,13 +59,18 @@ def bullet_client(TIMESTEP_SEC=time_step):
 
 
 @pytest.fixture
-def vehicle(bullet_client, TIMESTEP_SEC=time_step):
+def vehicle(bullet_client, vehicle_controller_file, TIMESTEP_SEC=time_step):
```

Comment:
I don't see where it's been updated to pass these new params in?

Reply:
Thanks, it is inside the `controller_parameters.yaml` inside the following lines:

```
with pkg_resources.path(models, "controller_parameters.yaml") as controller_path:
        controller_filepath = str(controller_path.absolute())
    with open(controller_filepath, "r") as controller_file:
        VEHICLE_CONTROLLER_PARAMETERS = yaml.safe_load(controller_file)[request.param]
```

Reply:
I mean where you call vehicle(...)

Reply:
`vehicle_controller_file` is also a fixture.